data/json/npcs/exodii/exodii_merchant_talk.json

```
@@ -571,11 +571,14 @@
   {
     "id": "TALK_EXODII_MERCHANT_Home",
     "type": "talk_topic",
-    "dynamic_line": "*becomes distant for a moment.*  \"Ah.  Home.  It were a long time gone, kennit.  Your'n be not too far.  I… don't think I want to talk 'bout it.  Too much metal atwixt now an' then.",
+    "dynamic_line": "*becomes distant for a moment.*  \"Ah.  Home.  It were a long time gone, kennit.  Your'n be not too far.  I… don't think I want to talk 'bout it.  Too much metal atwixt now an' then.\"",
```

```suggestion
    "dynamic_line": "*becomes distant for a moment.  \"Ah.  Home.  It were a long time gone, kennit.  Your'n be not too far.  This'n… I don't think I want to talk 'bout it.  Too much metal atwixt now an' then.\"",
```

Small typo and also reading my own writing back I think doing it like this helps emphasize Rubik's unusual switch to the first person.